### PR TITLE
Got cross and cross-local tests working on my box

### PR DESCRIPTION
**Pull request by ryanwilson-wk**
### Story:
Setting up test framework for working on https://jira.atl.workiva.net/browse/SPLAT-108

### Acceptance Criteria:
- [ ] At least one InfRe Squad 2 member has reviewed and +1'd
- [ ] Code has been tested and results documented
- [ ] Unit tests have been updated
- [ ] Updates to documentation if necessary
- [ ] Verify and document changes to any other Messaging components
- [ ] Pull request made against the 'develop' branch, not master

### Design Notes:
I tried to capture all the things I had to do in the `## Assumptions` section of the integration tests `readme.md`

### How To Test:
`wk test run cross-local`

I was using the wk tool to wrap the skynet CLI, either should work.

### My Test Results:
All tests passed

#### Reviewers:
@Workiva/service-platform

### CODE REVIEW

File: lib/go/go.mod [hunk 1]
```diff
@@ -5,8 +5,9 @@ require (
 	github.com/Sirupsen/logrus v0.11.5
 	github.com/go-stomp/stomp v2.0.6+incompatible
 	github.com/mattrobenolt/gocql v0.0.0-20130828033103-56c5a46b65ee
+	github.com/nats-io/gnatsd v1.4.1
 	github.com/nats-io/go-nats v0.0.0-20161120202126-6b6bf392d34d
-	github.com/nats-io/nats-server/v2 v2.1.7
+	github.com/nats-io/nats-server/v2 v2.1.7 // indirect
```
Comment (ryanwilson-wk) [inline lib/go/go.mod:10]: New to `go` and wasn't sure why this changed... I just ran `make`

**Comment by aviary-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://dev.workiva.net/docs/teams/information-security/aviary#configuring-aviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: scripts/skynet/cross/setup_java.sh [hunk 2]
```diff
@@ -9,7 +9,7 @@ if [ -z "${IN_SKYNET_CLI+yes}" ]; then
 else
     cd ${FRUGAL_HOME}/lib/java
     mvn clean verify -q
-    mv target/frugal-LOCAL.jar ${FRUGAL_HOME}/test/integration/java/frugal-integration-test/frugal.jar
+    mv target/frugal-3.10.0.jar ${FRUGAL_HOME}/test/integration/java/frugal-integration-test/frugal.jar
```
Comment (charliekump-wf) [inline scripts/skynet/cross/setup_java.sh:12]: Assume this needs to go back before this goes in? We don't want to hard code the tested version correct?

File: scripts/skynet/cross/setup_python.sh [hunk 3]
```diff
@@ -10,6 +10,7 @@ else
     cd $GOPATH/src/github.com/Workiva/frugal/lib/python
 fi
 
-pip install -e ".[tornado]"
-python3 /usr/bin/pip3 install Cython==0.27.3
-python3 /usr/bin/pip3 install -e ".[asyncio]"
+python2 -m pip install -e ".[tornado]"
```
Comment (charliekump-wf) [inline scripts/skynet/cross/setup_python.sh:13]: Sorry, this breaks for me 🤷 Seems the artifactory auth isn't working for me like it must be for you.

```
2020-09-18T16:10:31.504178200Z + python2 -m pip install -e '.[tornado]'
2020-09-18T16:10:31.504205100Z WARNING: Running pip install with root privileges is generally not a good idea. Try `__main__.py install --user` instead.
2020-09-18T16:10:31.504239100Z Obtaining file:///testing/lib/python
2020-09-18T16:10:31.504264200Z Collecting six<2,>=1.10.0 (from frugal==3.10.0)
2020-09-18T16:10:31.504290400Z Exception:
2020-09-18T16:10:31.504311700Z Traceback (most recent call last):
2020-09-18T16:10:31.504358800Z   File "/usr/lib/python2.7/site-packages/pip/basecommand.py", line 215, in main
2020-09-18T16:10:31.504381600Z     status = self.run(options, args)
2020-09-18T16:10:31.504405100Z   File "/usr/lib/python2.7/site-packages/pip/commands/install.py", line 346, in run
2020-09-18T16:10:31.504428500Z     requirement_set.prepare_files(finder)
2020-09-18T16:10:31.504461600Z   File "/usr/lib/python2.7/site-packages/pip/req/req_set.py", line 380, in prepare_files
2020-09-18T16:10:31.504475400Z     ignore_dependencies=self.ignore_dependencies))
2020-09-18T16:10:31.504506800Z   File "/usr/lib/python2.7/site-packages/pip/req/req_set.py", line 554, in _prepare_file
2020-09-18T16:10:31.504531200Z     require_hashes
2020-09-18T16:10:31.504554400Z   File "/usr/lib/python2.7/site-packages/pip/req/req_install.py", line 278, in populate_link
2020-09-18T16:10:31.504567900Z     self.link = finder.find_requirement(self, upgrade)
2020-09-18T16:10:31.504585100Z   File "/usr/lib/python2.7/site-packages/pip/index.py", line 465, in find_requirement
2020-09-18T16:10:31.504626100Z     all_candidates = self.find_all_candidates(req.name)
2020-09-18T16:10:31.504648600Z   File "/usr/lib/python2.7/site-packages/pip/index.py", line 423, in find_all_candidates
2020-09-18T16:10:31.504673300Z     for page in self._get_pages(url_locations, project_name):
2020-09-18T16:10:31.504707000Z   File "/usr/lib/python2.7/site-packages/pip/index.py", line 568, in _get_pages
2020-09-18T16:10:31.504873200Z     page = self._get_page(location)
2020-09-18T16:10:31.504939800Z   File "/usr/lib/python2.7/site-packages/pip/index.py", line 683, in _get_page
2020-09-18T16:10:31.504989000Z     return HTMLPage.get_page(link, session=self.session)
2020-09-18T16:10:31.505006100Z   File "/usr/lib/python2.7/site-packages/pip/index.py", line 792, in get_page
2020-09-18T16:10:31.505026500Z     "Cache-Control": "max-age=600",
2020-09-18T16:10:31.505053400Z   File "/usr/lib/python2.7/site-packages/pip/_vendor/requests/sessions.py", line 521, in get
2020-09-18T16:10:31.505083000Z     return self.request('GET', url, **kwargs)
2020-09-18T16:10:31.505120100Z   File "/usr/lib/python2.7/site-packages/pip/download.py", line 386, in request
2020-09-18T16:10:31.505147100Z     return super(PipSession, self).request(method, url, *args, **kwargs)
2020-09-18T16:10:31.505178200Z   File "/usr/lib/python2.7/site-packages/pip/_vendor/requests/sessions.py", line 508, in request
2020-09-18T16:10:31.505211600Z     resp = self.send(prep, **send_kwargs)
2020-09-18T16:10:31.505298700Z   File "/usr/lib/python2.7/site-packages/pip/_vendor/requests/sessions.py", line 625, in send
2020-09-18T16:10:31.505323000Z     r = dispatch_hook('response', hooks, r, **kwargs)
2020-09-18T16:10:31.505371700Z   File "/usr/lib/python2.7/site-packages/pip/_vendor/requests/hooks.py", line 31, in dispatch_hook
2020-09-18T16:10:31.505396500Z     _hook_data = hook(hook_data, **kwargs)
2020-09-18T16:10:31.505414600Z   File "/usr/lib/python2.7/site-packages/pip/download.py", line 181, in handle_401
2020-09-18T16:10:31.505447600Z     username = six.moves.input("User for %s: " % parsed.netloc)
2020-09-18T16:10:31.505482100Z EOFError: EOF when reading a line
2020-09-18T16:10:31.505572100Z User for workivaeast.jfrog.io:
2020-09-18T16:10:31.506355200Z exit status 1
2020-09-18T16:10:31.533659600Z + $ exit 1
2020-09-18T16:10:31.540734800Z [203] 2020/09/18 16:10:31.536740 [ERR] Failed to reload server configuration: can only reload config when a file is provided using -c or --config
Connection to localhost closed by remote host.
```

File: scripts/skynet/cross/setup_java.sh (hunk 2 above)
Comment (ryanwilson-wk) [reply scripts/skynet/cross/setup_java.sh:12]: There was no artifact with the version `LOCAL` prior to this change, so it was always failing. I would like to automate updating the version. Or maybe we can update this script to not be dependent on the version.

**Review by natewoods-wf — APPROVED** [review]
+1 having someone be able to test locally is better than nobody being able to test locally, going into an integration branch where we can later review + augment with changes to work for all devs

File: scripts/skynet/cross/setup_python.sh (hunk 3 above)
Comment (charliekump-wf) [reply scripts/skynet/cross/setup_python.sh:13]: Ah yeah, forgot about that. One of the many reasons I do not like the Skynet CLI

**Comment by natewoods-wf** [comment]
@Workiva/release-management-p for merge

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Pull request merged**